src/app/pubspec.lock

```diff
@@ -564,10 +572,10 @@ packages:
     dependency: "direct main"
     description:
       name: shared_preferences
-      sha256: "81429e4481e1ccfb51ede496e916348668fd0921627779233bd24cc3ff6abd02"
+      sha256: d3bbe5553a986e83980916ded2f0b435ef2e1893dfaa29d5a7a790d0eca12180
       url: "https://pub.dev"
     source: hosted
-    version: "2.2.2"
+    version: "2.2.3"
```

Comment:
that's weird

Reply:
Probably just an update.

Reply:
yeah it's because I did the update by mistake but didn't revert it properly

Reply:
To be honest if it's not breaking anything update anything you want, also this is still in pre-release, it's all good.